docker-compose.yml

```diff
@@ -297,13 +289,12 @@ services:
       - shared:/shared
 
 volumes:
+  cacert:
   pgdata:
   shared:
   s3data:
 
 networks:
   public:
   secure:
-  # required for allowing auth to connect to ls-aai-mock
-  my-app-network:
-    name: my-app-network
```

Comment:
ls-aai mock will fail if it cannot find [this network](https://github.com/GenomicDataInfrastructure/starter-kit-lsaai-mock/blob/3d84eaf6e82bc034df3ebe497147e3a963ece617/docker-compose.yaml#L54)

Reply:
```
    extra_hosts:
    - dockerhost:host-gateway
```
This makes it possible to access the `ls-aai-mock` as `http://dockerhost:PORT`. Where `PORT` is the port you bind on the host machine, 8080 in the case of the compose file you linked to.